File: remote/src/main/java/com/kelsos/mbrc/ui/views/ArtistAlbumsView.java
package com.kelsos.mbrc.ui.views;

import com.kelsos.mbrc.domain.Album;
import java.util.List;

public interface ArtistAlbumsView {

  void update(List<Album> data);

  void showLoadFailed();

  void queueSuccess();

  void queueFailed();
}

File: app/src/main/kotlin/com/kelsos/mbrc/ui/navigation/library/artist_albums/ArtistAlbumsView.kt
package com.kelsos.mbrc.ui.navigation.library.artist_albums

import com.kelsos.mbrc.domain.Album

interface ArtistAlbumsView {

  fun update(data: List<Album>)

  fun showLoadFailed()

  fun queueSuccess()

  fun queueFailed()
}
